packages/office-addin-mock/src/officeMockObject.ts

```diff
@@ -82,6 +82,12 @@ export class OfficeMockObject {
     }
   }
 
+  private loadAllProperties() {
+    Array.from(this.properties.keys()).forEach((property: string) => {
```

Comment:
Do we need the 'Array.from' call?  Doesn't the "keys()' call already return an array?

Reply:
OK . . . had to remind myself of the objects at play here.  What about 'this.properties.foreach' as properties is a Map object that does have a foreach directly on it?  I'm not sure how much is matters or if one is way is more performant then others, I just thing it's easier to read.

Reply:
I just managed to iterate over the map using `forEach((value, key))` now. It should be more clear now.